### PR TITLE
S-94204 review changes

### PR DESCRIPTION
**Pull request by rohithamroser**

### CODE REVIEW

File: gitlab/config.json [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        {
+            "title": "Description",
+            "name": "description",
+            "type": "input"
+        }
+    ],
+    "assets_properties": [
+        {
+            "type":"custom",
+            "title":"organization",
```
Comment (DhilipKumar0807) [inline gitlab/config.json]: give space after colon

File: gitlab/mapping.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+            "LOOKUP": "lookup",
+            "RELATION": "Relation"
+        }
+        fields = {
+            "title": {
+                "type": fields_type["TEXT"],
+                "system": "title"
+            }, 
+            
+
```
Comment (DhilipKumar0807) [inline gitlab/mapping.py]: remove extra space line

File: gitlab/mapping.py [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+            },
+            "description": {
+                "type": fields_type["TEXT"],
+                "system": "description"
+            },
+            
+        }
+        
+        
+
```
Comment (DhilipKumar0807) [inline gitlab/mapping.py]: remove extra line spaces

File: gitlab/mapping.py [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+                    'id': str(project['name']),
+                    "project": str(self.query_params["organization"]["display_name"]) + "/" + str(project["id"]),
+                    'display_name': project['name'],
+                    
+                }
+            )
+        return projects
+    
+    def fetch_assets(self):
+        org = self.query_params
```
Comment (DhilipKumar0807) [inline gitlab/mapping.py]: is this org needed?

File: gitlab/sync.py [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+class Payload(BasePayload):
+
+    def fetch_project(self, event):
+        project = str(event["project"]["id"])
+        org = event['project']['namespace']
+        return org + "/" + project
+    
+    def fetch_asset(self, event):
+        return "Assettype-001"
```
Comment (DhilipKumar0807) [inline gitlab/sync.py:28]: Map asset and return value

Comment (rohithamroser) [reply gitlab/sync.py:28]: api dosnt provide value for the asset,so i gace the default name asserttype-001 for issue in gilab

**Comment by rohithamroser** [comment]
done

**Pull request merged**